pkgs/development/mobile/xcodeenv/default.nix

```diff
@@ -2,7 +2,7 @@
 
 rec {
   composeXcodeWrapper = import ./compose-xcodewrapper.nix {
```

Comment:
Why are we not using callPackage here?

Reply:
Absolutely no idea. I didn't write this. We can change it.